importer/mappings/P31_blacklist.json

```diff
@@ -106,5 +106,33 @@
     {
         "item": "Q4022",
         "comment": "river"
+    },
+    {
+        "item": "Q2061186",
+        "comment": "religious order"
+    },
+    {
+        "item": "Q43229",
+        "comment": "organization"
+    },
+    {
+        "item": "Q123705",
```

Comment:
I'm not sure about Q123705/neighbourhood. I think this is very often used for a chunk of a city in a particular style which could very well be protected.

Reply:
:white_check_mark: